content/_data/changelogs/weekly.yml

```diff
@@ -12207,16 +12207,24 @@
       pull: 5425
       authors:
       - daniel-beck
+      references:
+      - pull: 5425
+      - url: https://www.jenkins.io/doc/book/managing/built-in-node-migration/
+        title: Built-In Node Name and Label Migration
+      - url: https://www.jenkins.io/doc/book/security/controller-isolation/
+        title: Do not build on the built-in node
       message: |-
         Replace the term "master" with "controller" (for the main Jenkins application) or "built-in node", as appropriate.
+        References to the "master" label in job definitions, Pipeline definitions, tool installer should be moved to others agents where ever possible.
+        If a job definition, Pipeline definition, or tool installer reference must be tied to the built-in node, it should use the label "built-in".
```

Comment:
This advice seems largely unrelated to the terminology change (don't use the built-in node applied before and applies now), so doesn't belong in the changelog.

Besides that, the label change is the next changelog entry, so this one doesn't need changing IMO.

Reply:
Suggest rewording : 

Where a Job definition, Pipeline definition, or tool installer references label  "master" as the built-in node, it should be changed use label "built-in".  NB: Building on the built-in node is strongly discouraged (https://www.jenkins.io/doc/book/security/controller-isolation/).